gatsby/src/pages/index.tsx

```diff
@@ -21,7 +22,17 @@ const IndexPage: React.FC<IndexProps> = ({
 }: IndexProps) => {
 	return (
 		<Layout>
-			<SEO />
+			<SEO
+				additionalMetadata={
+					//we pass in a react helmet component here as due to this bug https://github.com/nfl/react-helmet/issues/342
+					<Helmet>
```

Comment:
If we're rendering an additional `Helmet` here, then there's no need to pass it into the `additionalMetadata` prop, we might as well just render it straight inside this component, which would make  `additionalMetadata` obsolete.

I sort of prefer the idea of keeping the meta stuff together (rather than rendering multiple helmets in different places as that could get hard to manage if there were lots). [This comment](https://github.com/nfl/react-helmet/issues/342#issuecomment-566542589) seems to be a nicer solution - the `metaData` could be a prop, typed with TypeScript so you could pass in:

```jsx
<SEO metaData={[{ name: "google-site-verification", content: "3N3Ng_4D9vTfn0AubNl1BjDivNeDmo_erefsd_ClwL4" ]]}
```